spotify-widget/spotify.lua

```diff
@@ -16,6 +16,10 @@ local GET_SPOTIFY_STATUS_CMD = 'sp status'
 local GET_CURRENT_SONG_CMD = 'sp current'
 
 local function ellipsize(text, length)
+    -- utf8 only available in Lua 5.3+
+    if utf8 == nil then
+        return text
```

Comment:
Maybe it would be better to trim it anyway? Like
```lua
return text:sub(0, length)
```

Reply:
good idea; fixed